packages/patternfly-3/react-console/src/DesktopViewer/ManualConnection.js

```diff
@@ -0,0 +1,79 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import consoleDetailPropType from './consoleDetailPropType';
+
+const Detail = ({ title, value }) => (
```

Comment:
@lizsurette do we have any design specs on providing connection details? Otherwise, I don't know if we need a standard component for this or not (it may just make use of other existing components downstream.

<img width="421" alt="screen shot 2018-09-21 at 3 47 39 pm" src="https://user-images.githubusercontent.com/4237045/45902842-cbc1b500-bdb5-11e8-99fd-752af319021f.png">

Reply:
@priley86 - @mareklibra might be able to speak to the specific use case for this, I could envision this being something we surface in the KubeVirt UI for VM Details. I'm not sure we need a specific component for it or if we should just place to have this detail as a part of the larger details view. Maybe this is being targeting for another use I'm not aware of though :)

Reply:
This is not Kubevirt-specific. This is yet-another way how to remotely connect to graphical interface of a VM or physical server.
The use-case is for an user with non-browser (desktop-based) viewer which is not accepting the `application/x-virt-viewer` mime-type.
It's the way how to use desktop-based viewer and not to be locked to the `remote-viewer` application.
I think not only Mac users can benefit from that but even Linux users.

Reply:
ok - thanks. This is useful to know and I see your reasoning. I also see that you can optionally provide those individual connection details. To avoid having a consumer need to make changes to PF React, I would think that hypothetically one might need to add additional connection details. For that reason, could we possible provide a `additionalDetails` array of key/value prop to `ManualConnection` to render beneath `textVNCTlsPort`. Additionally it may be nice to have a `children` prop to render beneath the `div` for additional info:

```
    <div className="manual-connection-pf">
      <h2>{textManualConnection}</h2>
      <p>{msg}</p>
      <div>...</div>
      {children}
   </div>
```
If you don't think this is useful and you feel it is this strict, feel free to ignore this advice 😄. I just see this one being very specific...